packages/common/lib/functions/TypeConversionUtils.ts

```diff
@@ -10,13 +25,19 @@ export const getNumberOrUndefined = (input?: string): number | undefined => {
  * @param subject
  */
 export function getTypesFromObject(
-  subject: CredentialConfigurationSupported | CredentialOfferFormat | CredentialDefinitionV1_0_13 | JsonLdIssuerCredentialDefinition | string,
+  subject:
+    | CredentialConfigurationSupported
+    | CredentialOfferFormat
+    | CredentialDefinitionJwtVcJsonLdAndLdpVcV1_0_13
+    | CredentialDefinitionJwtVcJsonV1_0_13
+    | JsonLdIssuerCredentialDefinition
+    | string,
 ): string[] | undefined {
   if (subject === undefined) {
     return undefined;
   } else if (typeof subject === 'string') {
     return [subject];
-  } else if ('credential_definition' in subject && subject.credential_definition) {
+  } else if ('format' in subject && subject.format !== 'vc+sd-jwt' && 'credential_definition' in subject) {
```

Comment:
I am not entirely sure whether format would always be present for this expression branch given the amount of types this function accepts. So probably better to change the logic around to search for credential_definition as long as there is no format or format not equal to sd-jwt. So something like !format or format !== sd-jwt'

Reply:
Hmm all the types that are supported for `subject` in the function parameter, either have no `credential_definition` defined, or will always have both `format` and `credential_definition` defined. 

But changed it now to not check format. This method is supposed to catch all, so being permissive is probably good.